build/gulp/tasks/bundle.ts

```diff
@@ -2,70 +2,73 @@ import { task, series, parallel, src, dest } from 'gulp'
 import * as merge2 from 'merge2'
 import * as rimraf from 'rimraf'
 import * as webpack from 'webpack'
+import { argv } from 'yargs'
 
 import config from '../../../config'
+import sh from '../sh'
 
 const { paths } = config
 const g = require('gulp-load-plugins')()
 const { log, PluginError } = g.util
 
+const packageName: string = argv.package || 'react'
+
 // ----------------------------------------
 // Clean
 // ----------------------------------------
 
-task('clean:dist:es', cb => {
-  rimraf(`${config.paths.dist()}/es/*`, cb)
+task('bundle:clean:es', cb => {
+  rimraf(`${config.paths.packageDist(packageName)}/es/*`, cb)
 })
 
-task('clean:dist:commonjs', cb => {
-  rimraf(`${config.paths.dist()}/commonjs/*`, cb)
+task('bundle:clean:commonjs', cb => {
+  rimraf(`${config.paths.packageDist(packageName)}/commonjs/*`, cb)
 })
 
-task('clean:dist:umd', cb => {
-  rimraf(`${config.paths.dist()}/umd/*`, cb)
+task('bundle:clean:umd', cb => {
+  rimraf(`${config.paths.packageDist(packageName)}/umd/*`, cb)
 })
 
-task('clean:dist', parallel('clean:dist:es', 'clean:dist:commonjs', 'clean:dist:umd'))
+task('bundle:clean', parallel('bundle:clean:es', 'bundle:clean:commonjs', 'bundle:clean:umd'))
```

Comment:
still a bit confused with the naming, sorry :) Here is the reason for that - in general the following structure for task names is suggested:

`bundle:package:...` - package specific tasks

At the same time we have

`bundle:clean` - and this one turns out to be package-specific as well (this moment is quite confusing)

`bundle:all-packages` - actually, we could safely rename it to `bundle:all`

### Proposal
As a proposal I might suggest something like the following (at the same time, please, feel free to introduce another approach - there are only aforementioned points I would like to be addressed):

```
bundle:package - builds all module resolution versions (I would include umd here)
bundle:package:no-umd - this one will be used in the build chain

bundle:package:clean
bundle:package:clean:umd

bundle:build:all (or bundle:build) - the sole reason I won't suggest to use 'bundle:all' is because the actual action taken as part of this task is different, so that 'bundle:all !== bundle:package for each package', according to the task's logic
```

Reply:
Updated all expect `bundle:all-packages`, it's very clear what task actually does